harness/determined/cli/trial.py

```diff
@@ -121,6 +126,77 @@ def trial_logs(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def generate_support_bundle(args: Namespace):
+    temp_dir = tempfile.TemporaryDirectory()
+    trial_logs_filepath = write_trial_logs(args, temp_dir)
+
+    master_logs_filepath = write_master_logs(args, temp_dir)
+
+    api_experiment_filepath, api_trail_filepath = write_api_call(args, temp_dir)
+
+    bundle = None
+    if args.output_dir is None:
+        output_dir = os.getcwd()
+
+    else:
+        bundle = tarfile.open(f"{args.output_dir}.tar.gz", "w:gz")
+
+    filename_printf = "det-bundle-trial-{trial_id}-{datetime}.tar.gz"
+    tar_filename = filename_printf.format(
```

Comment:
you can use f-strings. I think we're requiring at least 3.6 now.

Reply:
I used f-strings somewhere else and lint is complaining.